### ISSUE TITLE
init: Add --json flag to "minio" (enhancement)

### ISSUE THREAD

**Issue by yhafri** [minio/minio#3332]
This new flag **--json** will mimic the same option as in *mc*.
Instead of displaying raw data, format everything in _JSON_

Currently, we've:

```
$ minio server --address "127.0.0.1:8888" /tmp/.s3share

Endpoint: http://127.0.0.1:8888
AccessKey: KM6074YUX3OB5BK06D02
SecretKey: LuQwctJmvJQUyPA6ogYWKtkMLFo9uEBez6/5Y/o5
Region: us-east-1
SqsARNs:

Browser Access:
http://127.0.0.1:8888

Command-line Access: https://docs.minio.io/docs/minio-client-quickstart-guide
$ mc config host add myminio http://127.0.0.1:8888 KM6074YUX3OB5BK06D02 LuQwctJmvJQUyPA6ogYWKtkMLFo9uEBez6/5Y/o5
```
## Expected Behavior

With the *--json* flag in place, we get this instead:

```
$ minio server --json --address "127.0.0.1:8888" /tmp/.s3share

[ 
{"endpoint" : "http://127.0.0.1:8888"},
{"access_key" : "KM6074YUX3OB5BK06D02",
{"secret_key" : "LuQwctJmvJQUyPA6ogYWKtkMLFo9uEBez6/5Y/o5",
{"region" : "us-east-1"},
{"port", "8888"} 
...
]
```

## Current Behavior

The output information is very helpful and can be used for other purposes (see: [#3314](https://github.com/minio/minio/issues/3314)).

**Comment by deekoder** [comment]
Needs to be discussed.

**Comment by deekoder** [comment]
Work in progress by @kannappanr

**Comment by kannappanr** [comment]
Fixed by #5293
